peach_package_builder/build_peach_config.py

```diff
@@ -4,12 +4,26 @@
 import argparse
 import subprocess
 import sys
+import re
 import os
 
 from peach_package_builder.constants import *
 from peach_package_builder.utils import add_deb_to_freight, update_freight_cache
 
 
+def get_version_from_setup_file(file_path):
+    with open(file_path, 'r') as f:
+        lines = f.read().splitlines()
+        for line in lines:
+            print(line)
```

Comment:
The `print(line)` is just for debugging purposes?

Reply:
oops didn't mean to leave that there thanks! will delete